OJP_Common.xsd

```diff
@@ -72,6 +72,41 @@
 			<xs:element name="OperatorRef" type="siri:OperatorRefStructure" minOccurs="0" maxOccurs="unbounded"/>
 		</xs:sequence>
 	</xs:complexType>
+	<xs:annotation>
+		<xs:documentation>========================================= Type of Product Categories =============================================</xs:documentation>
+	</xs:annotation>
+	<xs:simpleType name="TypeOfProductCategoryType">
```

Comment:
I guess you mean TypeOfProductCategoryIdType (to reference a TypeOfProductCategory via its Id) ?